web/src/datastore.js

```diff
@@ -82,22 +80,31 @@ const generateId = props => () => props.firestore.collection("ids").doc().id;
 const getLocalizedText = obj => obj && (obj["_"] || obj["en"] || obj["pt"]);
 
 // Mount project data onto store based on current projectId in path.
-const connectGndDatastore = compose(
-	connect((store, props) => ({
-		projectId: getActiveProjectId(store)
-	})),
-	firestoreConnect(({ projectId }) => [
+const connectProject =
+	firestoreConnect((store, props) => [
 		{
 			collection: "projects",
-			doc: projectId,
+			doc: store.match.params.projectId,
 			storeAs: "activeProject"
 		},
 		{
-			collection: `projects/${projectId}/features`,
+			collection: `projects/${store.match.params.projectId}/features`,
 			storeAs: "mapFeatures"
 		}
-	])
-);
+	]);
+
+// Mount feature data onto store based on current projectId and featureId in
+// path.
+const connectFeature =
+	firestoreConnect((store, props) => [
+		{
+			collection: `projects/${store.match.params.projectId}/records`,
+      where: [
+         ['featureId', '==', store.match.params.featureId]
+      ],
+			storeAs: "featureRecords"
+		}
+	]);
```

Comment:
Nit: Please use two spaces for indentation:
https://google.github.io/styleguide/jsguide.html#formatting-block-indentation

Consider using Beautify for VSCode or equivalent in Atom, Sublime, or your favorite editor. Eventually we'll set up formatting requirements w/lint checks.

Reply:
so i used prettier and it resulted in more diffs then I expected

Reply:
in my atom it looks like tabs are being used... I just copy pasted what we in the lines that you added... 

Maybe I will try VSCode for next change..

Reply:
It still looks pretty funky. Could you update your settings and try again, manually if necessary?

Reply:
is it still funky? I did it by hand..